lib/swoosh/adapters/mailgun.ex

```diff
@@ -79,19 +79,23 @@ defmodule Swoosh.Adapters.Mailgun do
   end
   defp prepare_custom_vars(body, _email), do: body
 
+  defp prepare_attachments(body, %{attachments: []}), do: body
+  defp prepare_attachments(body, %{attachments: attachments}) do
+    Map.put(body, :attachment, Enum.map(attachments, &File.read!(&1.path)))
```

Comment:
Not wondering whose fault it is that you haven't looked at that code in a while. Sorry Po...